updated SCIM images (#15002)
docker/docker.github.io: docker-hub/scim.md

![scim-app-provisioning](images/scim-app-provisioning.png){:width="700px"}

> 6. Remove the unwanted attributes from your mappings and fields from the Profile editor.

Reviewer: In the preview, this is displaying as step 1 instead of 6.

Author: Thank you, updated!

> 6. Remove the unwanted attributes from your mappings and fields from the Profile editor.

Reviewer 2: Profile editor is used to modify custom user attributes, we don't need them to do that, they just need to remove them from the Attributes mappings
```suggestion
6. Remove all fields that are not supported from your Attributes Mappings.
```

And add this image a
![Untitled](https://user-images.githubusercontent.com/11965230/175658782-13cab083-9613-4c01-aa53-cc229f38751d.png)
s reference of the attributes we do support:

Author: Thank you, updated!
